src/components/text/WarningText.tsx

```diff
@@ -0,0 +1,19 @@
+import { ParagraphStylesMap } from "./Text.constans.tsx";
+import { WarningTextStyled } from "./Text.styles.ts";
+import { TextTypes } from "./Text.types.tsx";
+
+const WarningText = ({ variant = "small", children }: TextTypes) => {
```

Comment:
I'd definitely create a new variant of Text instead of repeating most of the component's code. 
You only set red color from what I can tell.

Reply:
added!